### PR TITLE
Render terms & conditions in settings

### PR DESCRIPTION
**Pull request by dbkr**
If configured in config file

Fixes https://github.com/vector-im/riot-web/issues/7219

### CODE REVIEW

File: src/components/structures/UserSettings.js [hunk 1]
```diff
@@ -921,6 +921,25 @@ module.exports = React.createClass({
         </div>;
     },
 
+    _renderTermsAndConditionsLinks: function() {
+        if (SdkConfig.get().terms_and_conditions_links) {
+            const tncLinks = [];
+            for (const tncEntry of SdkConfig.get().terms_and_conditions_links) {
```
Comment (bwindels) [inline src/components/structures/UserSettings.js]: No need to check if `terms_and_conditions_links` is set here?

Comment (bwindels) [reply src/components/structures/UserSettings.js]: Oops, you do above already :)

Comment (bwindels) [inline src/components/structures/UserSettings.js]: `map` would fit well here? But oh well ...

Comment (dbkr) [reply src/components/structures/UserSettings.js]: yeah true - I suppose it doesn't make a lot of difference

**Pull request merged**